### PR TITLE
[twitter] concurrent view count for live streams & username

### PR DESCRIPTION
**Pull request by sonmezberkay**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

yt-dlp currently returns `total_watched` value for live streams which is the number of viewerers seen the live stream from its beginning until its end. With the changes, it will be possible to reach the number of viewers viewing the live stream in desired moment, and that is coming from `total_watching` field.

And `uploader` field has the value of user's name, but `twitter_username`, the one starting with '@' which may be useful will be returned after changes.

Fixes #

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check all of the following options that apply:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details>

### CODE REVIEW

File: yt_dlp/extractor/periscope.py [hunk 1]
```diff
@@ -31,9 +31,11 @@ def _parse_broadcast_data(self, broadcast, video_id):
             'title': title,
             'timestamp': parse_iso8601(broadcast.get('created_at')),
             'uploader': uploader,
+            'twitter_username': broadcast.get('twitter_username'),
```
Comment (bashonly) [inline yt_dlp/extractor/periscope.py:34]: this is extracted in `TwitterBroadcastIE`, and `twitter_username` is not a valid info_dict field
```suggestion
```

**Review by bashonly — CHANGES_REQUESTED** [review]
Hi, ~~please complete the PR template or else this PR cannot be accepted~~

> - [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
> - [x] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

which one is true?

**Comment by sonmezberkay** [comment]
> Hi, ~please complete the PR template or else this PR cannot be accepted~
> 
> > * [x]  I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
> > * [x]  I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)
> 
> which one is true?

sorry, updated.

File: yt_dlp/extractor/periscope.py (hunk 1 above)
Comment (sonmezberkay) [reply yt_dlp/extractor/periscope.py:34]: where exactly? the user name wasn't returned before this addition. Couldn't see while testing it with live streams

Comment (bashonly) [reply yt_dlp/extractor/periscope.py:34]: https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/twitter.py#L1620

your branch appears to be outdated

Comment (sonmezberkay) [reply yt_dlp/extractor/periscope.py]: Thanks, updated 👍

**Review by bashonly — APPROVED** [review]

**Pull request merged**